discord/ui/select.py

```diff
@@ -239,6 +256,52 @@ def from_component(cls, component: SelectMenu) -> Self:
             row=None,
         )
 
+    @staticmethod
+    def _handle_select_defaults(
```

Comment:
I don't like how this method basically discards the `value_type` information. e.g. If you pass a list of Role to a User select it doesn't tell you anything.